nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java

```diff
@@ -148,7 +154,17 @@ public static ColdStorageContentStatus checkAvailabilityOfColdStorageContent(Cor
         for (DocumentModel doc : documents) {
             if (isColdStorageContentAvailable(doc)) {
                 available++;
+                // mark the cold storage content of document as retrieved
+                doc.setPropertyValue(COLD_STORAGE_BEING_RETRIEVED_PROPERTY, false);
+                session.saveDocument(doc);
+
+                // create and fire the event
                 DocumentEventContext ctx = new DocumentEventContext(session, session.getPrincipal(), doc);
+                Map<String, Serializable> properties = new HashMap<>();
+                // FIXME: wait for NXP-28417
+                properties.put(NUMBER_OF_DAYS_OF_AVAILABILITY_KEY, "TO DEFINE");
+                properties.put(ARCHIVE_LOCATION_KEY, "https://www.nuxeo.com/");
```

Comment:
Maybe something generic like "TO_DEFINE" as well?

Reply:
you means just adding `TO_DEFINE` instead of `"https://www.nuxeo.com/"` or a new class filed `TO_DEFINE = "This will be defined"` and use it on the same

Reply:
second option could be fine